cbpi/controller/basic_controller2.py

```diff
@@ -87,7 +87,7 @@ async def start(self, id):
         logging.info("{} Start Id {} ".format(self.name, id))
         try:
             item = self.find_by_id(id)
-            if item.instance is not None and item.instance.state is True:
+            if item.instance is not None and item.instance.running is True:
```

Comment:
if my guess here was wrong this could potentially break a lot. please take a look.

My guess was that this is for duplicate prevention as suggested by the logging text. But the item.instance.state is used to store on/off state of an actor for example and not the start/stopped state.